app/services/EnrolmentService.scala

```diff
@@ -58,6 +58,12 @@ class EnrolmentService @Inject() (journeyService: JourneyService, auditService:
           EnrolmentDef.Sa.findEnrolmentValues(enrolments),
           journey
         )(journeyService.UpdateTaxRef.updateSaTaxId(journey.id, _))
+
+      case TaxRegime.Sia =>
```

Comment:
this could blow up if we want if we make sure we don't call this service for SIA